README instructions incorrect.

In attempting to follow the instructions on the README.md I'm unable to build the updated protos:

```
hoverbear@nomad:git/kvproto ‹master›$ make
mkdir -p /home/hoverbear/git/kvproto/bin
# Standalone GOPATH
./generate_go.sh
+ check_protoc_version
++ protoc --version
+ version='libprotoc 3.6.1'
++ echo libprotoc 3.6.1
++ sed -n -e 's/.*\([0-9]\{1,\}\)\.[0-9]\{1,\}\.[0-9]\{1,\}.*/\1/p'
+ major=3
++ echo libprotoc 3.6.1
++ sed -n -e 's/.*[0-9]\{1,\}\.\([0-9]\{1,\}\)\.[0-9]\{1,\}.*/\1/p'
+ minor=6
+ '[' 3 -gt 3 ']'
+ '[' 3 -eq 3 ']'
+ '[' 6 -ge 1 ']'
+ return 0
++ basename ./generate_go.sh
+ PROGRAM=generate_go.sh
+ '[' -z /home/hoverbear/go ']'
+ GO_PREFIX_PATH=github.com/pingcap/kvproto/pkg
+ gogo_protobuf_url=github.com/gogo/protobuf
++ pwd
+ CURRENT_DIR=/home/hoverbear/git/kvproto
+ GOGO_ROOT=/home/hoverbear/git/kvproto/vendor/github.com/gogo/protobuf
+ GO_OUT_M=
+ GO_INSTALL='go install'
+ echo 'install gogoproto code/generator ...'
install gogoproto code/generator ...
+ cd /home/hoverbear/git/kvproto/vendor/github.com/gogo/protobuf/protoc-gen-gofast
+ go build -o /home/hoverbear/git/kvproto/bin/protoc-gen-gofast
main.go:32:2: cannot find package "github.com/gogo/protobuf/vanity" in any of:
        /usr/lib/go/src/github.com/gogo/protobuf/vanity (from $GOROOT)
        /home/hoverbear/go/src/github.com/gogo/protobuf/vanity (from $GOPATH)
main.go:33:2: cannot find package "github.com/gogo/protobuf/vanity/command" in any of:
        /usr/lib/go/src/github.com/gogo/protobuf/vanity/command (from $GOROOT)
        /home/hoverbear/go/src/github.com/gogo/protobuf/vanity/command (from $GOPATH)
make: *** [Makefile:13: go] Error 1
```

I think this is because the repo is not in `GOPATH`? Since it is a requirement it should be detailed.

--- a/README.md
+++ b/README.md
@@ -19,13 +19,14 @@ Protocol buffer files for TiKV
 
 # Multiple `protoc` Versions
 
-If you need to override your version of `protoc` because you have a later version you can do the following instead of `make` below:
+If you need to override your version of `protoc` because you have a later version you can install the correct version like so:
 
 ```bash
 PROTOC_VERSION=3.1.0
-curl -L https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip -o protoc.zip
-unzip -p protoc.zip bin/protoc > protoc
+curl -L https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip -o protoc.zip &&\
+unzip protoc.zip -d protoc &&\
 rm protoc.zip
-chmod +x protoc
-PATH="`pwd`:$PATH" make
 ```
+
+Then you can run `PATH="$(pwd)/protoc/bin:$PATH" make`
+

--- a/README.md
+++ b/README.md
@@ -8,20 +8,24 @@ Protocol buffer files for TiKV
 * Protoc 3.1.0
 * This repo correctly placed in your `$GOPATH`.
 
-If you need to override your version of `protoc` because you have a later version you can do the following instead of `make` below:
-
-```bash
-PROTOC_VERSION=3.1.0
-curl -L https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip | unzip -p - bin/protoc > protoc-3.1.0
-PATH=".:$PATH" make
-```
-
 # Usage
 
 + Write your own protocol file in proto folder.
 + If you need to update raft-rs, please download the proto file
     respectively and overwrite the one in include folder.
-+ Run `make` to generate go and rust code. 
++ Run `make` to generate go and rust code.
     We generate all go codes in pkg folder and rust in src folder.
 + Update the dependent projects.
 
+# Multiple `protoc` Versions
+
+If you need to override your version of `protoc` because you have a later version you can do the following instead of `make` below:
+
+```bash
+PROTOC_VERSION=3.1.0
+curl -L https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip -o protoc.zip
+unzip -p protoc.zip bin/protoc > protoc
+rm protoc.zip
+chmod +x protoc
+PATH="`pwd`:$PATH" make
+```

--- a/README.md
+++ b/README.md
@@ -1,6 +1,21 @@
 # kvproto
 Protocol buffer files for TiKV
 
+# Dependencies
+
+* Rust
+* Go
+* Protoc 3.1.0
+* This repo correctly placed in your `$GOPATH`.
+
+If you need to override your version of `protoc` because you have a later version you can do the following instead of `make` below:
+
+```bash
+PROTOC_VERSION=3.1.0
+curl -L https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip | unzip -p - bin/protoc > protoc-3.1.0
+PATH=".:$PATH" make
+```
+
 # Usage
 
 + Write your own protocol file in proto folder.

